Source/Core/Core/HW/DSPHLE/UCodes/AX.h

```diff
@@ -141,6 +142,33 @@ class AXUCode : public UCodeInterface
   virtual void HandleCommandList();
   void SignalWorkEnd();
 
+  struct BufferDesc
+  {
+    int* ptr;
+    int samples_per_milli;
+  };
+
+  template<int Millis, size_t BufCount>
+  void InitMixingBuffers(u32 init_addr, const std::array<BufferDesc, BufCount>& buffers)
+  {
+    std::array<u16, 3 * BufCount> init_array;
+    Memory::CopyFromEmuSwapped(init_array.data(), init_addr, sizeof(init_array));
+    for (int i = 0; i < (int)BufCount; ++i)
```

Comment:
Any reason to have the case here instead of making `i` a size_t? As far as I can tell that would work